src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackSerializerOptions.cs

```diff
@@ -250,8 +251,8 @@ protected virtual MessagePackSerializerOptions Clone()
 #if !DYNAMICCODEDUMPER
         private static class MessagePackSerializerOptionsDefaultSettingsLazyInitializationHelper
         {
-            public static readonly MessagePackSerializerOptions Standard = new MessagePackSerializerOptions(Resolvers.StandardResolver.Instance, useLZ4Compression: false);
-            public static readonly MessagePackSerializerOptions LZ4Standard = Standard.WithLZ4Compression(true);
+            public static readonly MessagePackSerializerOptions Standard = new MessagePackSerializerOptions(Resolvers.StandardResolver.Instance);
+            public static readonly MessagePackSerializerOptions LZ4Standard = Standard.WithCompression(MessagePackCompression.LZ4Block);
```

Comment:
Should we remove or rename `LZ4Standard` since it may not be the only LZ4 scheme in the future, and may not even be the *recommended* scheme if/when we have better ones?

Reply:
yes, I think LZ4Standard should be deleted.

Perhaps the proposed LZ4ContiguousBlock is better in v2 and want to be used by default, but in any case it seems more natural for users to create with WithCompression each time.